install.sh

```diff
@@ -141,6 +141,10 @@ then
     bin/start.sh
 fi
 
+# if [-d "susi_skill_data"]
```

Comment:
Why does the media discovery daemon depend on the existence f susi_skill_data?
The daemon shall run additionally to any skills that susi has.

Reply:
@Orbiter ,this was done because the custom skills will be made in the local skill directory and if the skill data will not be present then the `autostart.sh` will not be able to execute properly.

Reply:
no the path to the skills for the media skills must be placed in `<susi_home>/data/generic_skills/media_discovery/`

This will be generated automatically by the susi server.
Do not write into `susi_skill_data`!

Reply:
okay @Orbiter , I'll fix this issue